backend/api/views.py

```
@@ -1,75 +1,163 @@
 from django.contrib.auth import login
-from .models import ViewerProfile, User
-from .serializers import OnboardingSerializer
-from django.http import HttpResponseRedirect
+from django.utils import timezone
+from .models import ViewerProfile, User, Video
+from .serializers import OnboardingSerializer, FirebaseTokenSerializer, VideoSerializer, VideoFeedSerializer, VideoDetailSerializer 
+from azure.storage.blob import generate_blob_sas, BlobSasPermissions
 
-class TestAuthView(APIView):
+class TestAuthView(generics.RetrieveAPIView):
     permission_classes = [IsAuthenticated]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)